engine/src/main/java/org/terasology/logic/console/commands/CoreCommands.java

```diff
@@ -83,15 +84,10 @@
 
 import java.io.IOException;
 import java.net.UnknownHostException;
+import java.util.*;
```

Comment:
One of our style rules is to avoid star imports. IntelliJ defaults to using them however.

You'll need to go into settings and change the number of imports required before a star is used up to 99. It's pain that we can't automatically do that at the moment but oh well